src/lib/game.ts

```diff
@@ -212,8 +212,8 @@ export function Game (mainCanvas: HTMLCanvasElement, skier: Skier) {
     runningTime = 0
   }
 
-  this.step = (now: number) => {
     if (paused) return
+  this.step = (now: number = performance.now()) => {
 
```

Comment:
Why are the lines swapped?

Reply:
Wow you are fast. I just saw it as well. the formatting confused me during the commit. I added a commit to just format the whole file, I hope that's okay.